app/Http/Controllers/AppointmentController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\PatientReserveAppointmentRequest;
+use App\Services\AppointmentService;
+use App\Services\DoctorService;
+use App\Services\PatientService;
+use App\Transformers\CreatedResource;
+use Illuminate\Http\Response;
+
+class AppointmentController extends Controller
+{
+    protected $appointmentService;
+
+    public function __construct(AppointmentService $appointmentService)
+    {
+        $this->appointmentService = $appointmentService;
```

Comment:
It's a better idea to name the service as just `service`

Reply:
done